client/components/MorphologyViewer/index.js

```diff
@@ -21,8 +21,8 @@ class MorphologyContainer extends React.Component {
     if (morphologySrc) {
       // TODO: refactor after demo
       // if swc, parse
-      const extension = morphologySrc.split('.');
-      if (extension[extension.length - 1] === 'swc' ) {
+      const extension = morphologySrc.split(".");
```

Comment:
Wouldn't it be better to use String.prototype.endsWith? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/endsWith